scripts/check_docs.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
+"""
+Check that alls functions are documented.
+"""
+import os
+import sys
+
+from typed_functions import all_functions
```

Comment:
as typed_functions is only used from this script, could you put the function in the script ?

Reply:
Done.